app/main.py

```diff
@@ -1,8 +1,57 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            comfort_class: int,
+            clean_mark: int,
+            brand: str
+    ) -> None:
+        self.comfort_class = (comfort_class
+                              if 1 <= comfort_class <= 7 else 1)
+        self.clean_mark = (clean_mark
+                           if 1 <= clean_mark <= 10 else 1)
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            distance_from_city_center: float,
+            clean_power: int,
+            average_rating: float,
+            count_of_ratings: int
+    ) -> None:
+        self.distance_from_city_center = (
+            distance_from_city_center
+            if 1.0 <= distance_from_city_center <= 10.0 else 1.0
+        )
+        self.clean_power = (clean_power
+                            if 1 <= clean_power <= 10 else 1)
+        self.average_rating = (average_rating
+                               if 1.0 <= average_rating <= 5.0 else 1.0)
+        self.count_of_ratings = count_of_ratings
```

Comment:
we never leave a part of instruction on the same lvl that the opening parenthesis. You should move the whole structure on te next line